<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-TA1D4P3" failures="0" tests="4" name="testNGProject.ExtentReportExample" time="23.345" errors="1" timestamp="2024-02-05T16:57:03 IST" skipped="3">
  <testcase classname="testNGProject.ExtentReportExample" name="launchBrowser" time="23.342">
    <error message="io/reactivex/rxjava3/subjects/PublishSubject" type="java.lang.NoClassDefFoundError">
      <![CDATA[java.lang.NoClassDefFoundError: io/reactivex/rxjava3/subjects/PublishSubject
at com.aventstack.extentreports.ReactiveSubject.<init>(ReactiveSubject.java:28)
at com.aventstack.extentreports.AbstractProcessor.<init>(AbstractProcessor.java:19)
at com.aventstack.extentreports.ExtentReports.<init>(ExtentReports.java:55)
at testNGProject.ExtentReportExample.launchBrowser(ExtentReportExample.java:30)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:69)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:390)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:325)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:382)
at org.testng.SuiteRunner.run(SuiteRunner.java:336)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1280)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1200)
at org.testng.TestNG.runSuites(TestNG.java:1114)
at org.testng.TestNG.run(TestNG.java:1082)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: java.lang.ClassNotFoundException: io.reactivex.rxjava3.subjects.PublishSubject
at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:641)
at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:188)
at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
... 21 more
]]>
    </error>
  </testcase> <!-- launchBrowser -->
  <testcase classname="testNGProject.ExtentReportExample" name="openBing" time="0.001">
    <skipped/>
  </testcase> <!-- openBing -->
  <testcase classname="testNGProject.ExtentReportExample" name="openWikipedia" time="0.001">
    <skipped/>
  </testcase> <!-- openWikipedia -->
  <testcase classname="testNGProject.ExtentReportExample" name="openGoogle" time="0.001">
    <skipped/>
  </testcase> <!-- openGoogle -->
</testsuite> <!-- testNGProject.ExtentReportExample -->
